### DISCUSSION TITLE
[Ethereum] Allow setting custom gas prices for users willing to wait during network congestion

### DISCUSSION THREAD

**Discussion by originalhamster** [LedgerHQ/ledger-live-desktop#3668] [Ideas]
As the title says, this is an enhancement request to allow users to set a custom gas price during ethereum network congestion or any scenario where they are willing to wait longer for block confirmation. Currently ledger users are forced to use third party software like MEW to gain this functionality.

In order to avoid confusion for unfamiliar users this can be implemented via an option that is enabled/disabled through LL settings.

**Comment by advanture-space** [comment]
Ledger currently allows custom gas prices but there is an enforced high threshold.

I am absolutely willing to wait. Some transactions I am willing to wait 1 week or even 1 months because it's very very low priority transactions.

Please let me set a custom low gas fee.
